transaction_items.go

```diff
@@ -0,0 +1,54 @@
+package dynago
+
+import (
+	"context"
+	"log"
+
+	"github.com/aws/aws-sdk-go-v2/feature/dynamodb/attributevalue"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb/types"
+)
+
+func (t *Client) WithDeleteItem(ctx context.Context, pk string, sk string) types.TransactWriteItem {
```

Comment:
context can be removed, these are not expected to do operations.

Reply:
yep addressed